backend/src/api/urls.py

```diff
@@ -1,10 +1,12 @@
 from django.contrib import admin
 from django.urls import path, include, re_path
-from .views import NewsViewSet, TagViewSet
+from .views import NewsViewSet, TagViewSet, TwitterAccountViewSet
 from rest_framework.routers import DefaultRouter
 
 router = DefaultRouter()
 router.register(r'news', NewsViewSet, basename='news')
-router.register(r'tag', TagViewSet, basename='tag') 
+router.register(r'tag', TagViewSet, basename='tag')
+router.register(r'twitterAccount', TwitterAccountViewSet, basename='twitterAccount') 
```

Comment:
Do we need an endpoint for it? We can make changes through django admin panel
If not then serializer and viewset is also not necessary

If you want to keep it, REST best practice would be to make the url
twitter-accounts
(and "tags" also)
https://restfulapi.net/resource-naming/

Reply:
True that